salt/cloud/clouds/profitbricks.py

```diff
@@ -120,7 +120,7 @@
     import profitbricks
     from profitbricks.client import (
         ProfitBricksService, Server,
-        NIC, Volume, FirewallRule,
+        NIC, Volume, FirewallRule, IPBlock,
```

Comment:
Is `IPBlock` new to the latest release of profitbricks? The minimum version required is `3.1.0` according to the docs in this file. This import needs to be handled more elegantly if someone is using profitbricks with an older version.